modern/clang/Dockerfile

```diff
@@ -247,7 +247,7 @@ RUN apt-get -qq update \
     && cp -a lib/clang/${CLANG_VERSION}/include /tmp/install/lib/clang/${CLANG_VERSION}/include \
     && cp $(find lib -name "*.so*") /tmp/install/lib
 
-FROM ${DOCKER_USERNAME}/gcc${LIBSTDCPP_MAJOR_VERSION}-${DISTRO}${SUFFIX}:${DOCKER_TAG} as libstdcpp
+FROM docker.io/conanio/gcc${LIBSTDCPP_MAJOR_VERSION}-${DISTRO}:latest as libstdcpp
```

Comment:
```suggestion
FROM docker.io/${DOCKER_USERNAME}/gcc${LIBSTDCPP_MAJOR_VERSION}-${DISTRO}:latest as libstdcpp
```

Reply:
we can't use DOCKER_USERNAME because it will use Artifactory instead. We do a push to Artifactory for cache images, with names like `docker push center-c3i-docker.jfrog.io/gcc10-ubuntu16.04-builder:master-n23`, then, we push to hub.docker.com with the final.

Reply:
but the value of DOCKER_USERNAME is `conanio`, right? So that will be `dcoker.io/conanio`

Reply:
@danimtb please, take a look in #565